src/nrc/tests/test_handle_auth_notifications.py

```diff
@@ -43,7 +41,9 @@ def test_handle_create_auth(self):
             "aanmaakdatum": "2012-01-14T00:00:00Z",
             "kenmerken": {},
         }
-        with mock_client(responses):
+
+        with requests_mock.Mocker(real_http=True) as m:
```

Comment:
`real_http` is used as there is an extra request made to get the OAS spec somewhere, and it is a pain to mock

Reply:
We have a special function from `zgw_consumers.test.mock_service_oas_get` to do it
It should looks something like this:

```
from zgw_consumers.test import mock_service_oas_get

mock_service_oas_get(m, url=config.api_root, service=APITypes.ac)
```

Reply:
It is being removed in https://github.com/maykinmedia/zgw-consumers/blob/main/CHANGELOG.rst#0330-2024-03-29 :/

Reply:
Then it looks like `zgw_consumers` and `zds_client` have conflicting versions now: `zds_client` still uses OAS yaml files and `zgw_consumers` removed the support for it. 
It should be either a downgrade of `zgw_consumers` or upgrade of `zds_client`to 2.0

Reply:
I haven't upgrade to 0.33.0 but 0.27.0. What I meant is that I don't know if it is a good idea to use a function that will be (considering we are currently on 0.27) deprecated. But thinking about it, I think it is fine to use it, as this could be tackled in another PR migrating from `zds_client` to the `ape_pie` clients (I haven't done it here as it is too much work and the priority is making it to Django 4.2)

Reply:
I think this is the way. We already use `mock_service_oas_get` multiple times in Open Notificaties, and moving to `ape-pie` will affects a lot of codebase, so it should be down as a separate thing.